runtime/js/99_main.js

```diff
@@ -850,9 +845,6 @@ function bootstrapMainRuntime(runtimeOptions, warmup = false) {
         nodeDebug,
       });
     }
-    if (internals.future) {
-      delete globalThis.window;
```

Comment:
Remove `window` from `98_global_scope_window.js`; you probably want to leave the `Window` interface though

Reply:
CC @lucacasonato am I correct here?